include/kernel.h

```diff
@@ -474,9 +474,7 @@ typedef struct _thread_base _thread_base_t;
 #if defined(CONFIG_THREAD_STACK_INFO)
 /* Contains the stack information of a thread */
 struct _thread_stack_info {
-	/* Stack Start - Identical to K_THREAD_STACK_BUFFER() on the stack
-	 * object. Represents thread-writable stack area without any extras.
-	 */
+	 /* Represents thread-writable stack area without any extras */
```

Comment:
The comment is more suitable for the documentation of the whole thread_stack_info struct
As is. it refers to the `start `variable.
 
Therefore, you could perhaps update it as "Represents the start address of the thread-writable stack area..."

Reply:
done